CMakeLists.txt

```diff
@@ -48,6 +45,14 @@ find_package(Boost REQUIRED COMPONENTS
 add_library(Common INTERFACE)
 add_library(VexCL::Common ALIAS Common)
 
+target_compile_features(Common INTERFACE
```

Comment:
Is there list of features that cmake knows about? I can see [here](https://cmake.org/cmake/help/v3.1/manual/cmake-compile-features.7.html) that we may want to add `cxx_variadic_templates`.

Reply:
Sure, [here](https://cmake.org/cmake/help/v3.1/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html#prop_gbl:CMAKE_CXX_KNOWN_FEATURES). I think I linked it before somewhere. You can use the dropdown at the top to pick CMake versions.

Reply:
Thanks! From the top of my head I can only see `cxx_decltype` there.